topbeat/beater/config.go

```diff
@@ -1,17 +1,48 @@
 package beater
 
+import (
+	"errors"
+	"time"
+)
+
+type ConfigSettings struct {
```

Comment:
Not so nice, as topbeat is special. In [topbeat.go:42](https://github.com/elastic/beats/pull/1659/files#diff-478f3b0d71959277c6d22459d829d90aR42) we do the unpack (as 'input' is deprectated alias for 'topbeat'). Maybe in v6 when removing support for deprecated sections we can make libbeat/beater pass correct section.